discovery-infra/start_discovery.py

```diff
@@ -300,6 +313,17 @@ def main():
         nodes_flow(client, cluster_name, cluster)
 
 
+def str2bool(v):
```

Comment:
what is it? just pass 'y' or 'n' , u handle it in Makefile why u need to have conversions? just pass relevant string

Reply:
It is more capable :)

Reply:
it is not

Reply:
use at least distutils.util.strtobool